marimo/_cli/convert/markdown.py

```diff
@@ -2,12 +2,11 @@
 from __future__ import annotations
 
 import re
-from typing import Any, Callable, Literal
+from typing import Any, Callable, Literal, Optional
 
 # Native to python
 from xml.etree.ElementTree import Element, SubElement
 
-# Note: yaml is also a python builtin
 import yaml
```

Comment:
i might have lazy imported this, just fyi, if you are not up-to-date with `main`

Reply:
I think in exporter, not here

Reply:
oh thats right